lib/onelogin/ruby-saml/slo_logoutresponse.rb

```diff
@@ -25,6 +40,13 @@ def create(settings, request_id = nil, logout_message = nil, params = {})
         @logout_url = settings.idp_slo_target_url + response_params
       end
 
+      # Creates the Get parameters for the logout response.
+      # @param settings [OneLogin::RubySaml::Settings|nil] Toolkit settings
+      # @param request_id [String] The ID of the LogoutRequest sent by this SP to the IdP. That ID will be placed as the InResponseTo in the logout response
+      # @param logout_message [String] The Message to be placed as StatusMessage in the logout response
+      # @param params [Hash]  Some parameters to build the logout response
+      # @return [Hash] Parameters
```

Comment:
what kind of `Parameters` ? is it possible we get more detail about the return value?